### ISSUE TITLE
Create an about dialog for all LXQt apps

### ISSUE THREAD

**Issue by ecsuarez** [lxqt/liblxqt#300]
I see wich any application of LXQt have they own about dialog, i think wich with a general about dialog is more attractive  and easy for the developer.

**Comment by tsujan** [comment]
Not all LXQt apps depend on liblxqt (QTerminal doesn't).

**Comment by ecsuarez** [comment]
Yes, but is good idea have one for an application wich need one, I can do it!
